### ISSUE TITLE
Number of failed image downloads

### ISSUE THREAD

**Issue by desb42** [gnosygnu/xowa#402]
I have just finished enwiki 2019-03-01 image downloading
This was based on images from 2018-07 (I think) so the total number of images that was attempted was approx 630,000
Of that there were 1824 failed downloads
Is this normal?

**Comment by gnosygnu** [comment]
Yeah, 1824 is sort of normal. Missing images usually fall into these categories
* Images deleted after the dump
* Images which just had bad file names

**Comment by gnosygnu** [comment]
Sorry. Premature comment. If you want, you can upload the err.txt here and I can do a quick once-over. Otherwise, I'll see it on my side sometime in the coming week.

Thanks!

**Comment by desb42** [comment]
[err.zip](https://github.com/gnosygnu/xowa/files/3000534/err.zip)
